### PR TITLE
Remove COSMO_TESTENV on arolla/tsa

### PR DESCRIPTION
**Pull request by lxavier**
Set machine name to tso

### CODE REVIEW

File: submit.tsa.slurm [hunk 1]
```diff
@@ -23,11 +23,8 @@ export MALLOC_TRIM_THRESHOLD_=536870912
 unset G2G
 export MV2_USE_GPUDIRECT=0 #with this perf are ok
 
-if [ ! -z $COSMO_TESTENV ] ; then
-  # UCX segfaults
-  export UCX_MEMTYPE_CACHE=n
-fi
-
+# UCX segfaults
+export UCX_MEMTYPE_CACHE=n
```
Comment (yboetz) [inline submit.tsa.slurm]: I think this is not necessary anymore, since this is exported in the modules file in env.tsa.sh

File: submit.tsa.slurm [hunk 2]
```diff
@@ -23,11 +23,8 @@ export MALLOC_TRIM_THRESHOLD_=536870912
 unset G2G
 export MV2_USE_GPUDIRECT=0 #with this perf are ok
```
Comment (yboetz) [inline submit.tsa.slurm]: Can we not remove this, since we're not using mvapich anymore?

File: env.tsa.sh [hunk 3]
```diff
@@ -78,8 +77,8 @@ setupDefaults()
     if [ -z "${fcompiler_cmd}" ] ; then
         if [ "${compiler}" == "gnu" ] ; then
             fcompiler_cmd="gfortran"
-         else
-            fcompiler_cmd="ftn"
+        else
+            fcompiler_cmd="pgfortran"
```
Comment (yboetz) [inline env.tsa.sh]: Maybe both of these should be changed to `mpif90`?

File: env.tsa.sh [hunk 4]
```diff
@@ -91,7 +90,7 @@ get_fcompiler_cmd()
     if [ "${compiler}" == "gnu" ] || [ "${compiler}" == "claw-gnu" ]; then
         myresult="gfortran"
     else
-        myresult="ftn"
+        myresult="pgfortran"
```
Comment (yboetz) [inline env.tsa.sh]: Are we not using `mpif90` in any case now?

File: env.tsa.sh [hunk 5]
```diff
@@ -46,18 +45,18 @@ setupDefaults()
...
-    export BASE_MODULES="craype-haswell"
+    export BASE_MODULES="craype-x86-skylake"
     export NVIDIA_CUDA_ARCH="sm_70"
 
     # BOOST
     export Boost_NO_SYSTEM_PATHS=true
     export Boost_NO_BOOST_CMAKE=true
 
-    export BOOST_ROOT=/project/c14/install/kesch/boost/boost_1_67_0/
+    export BOOST_ROOT=/project/c14/install/tsa/boost/boost_1_67_0/
```
Comment (elsagermann) [inline env.tsa.sh]: Is this path planned to be made? Because it doesn't exist yet!

Comment (yboetz) [reply env.tsa.sh]: All libraries/tools should be installed in the new `install/tsa` dir (using jenkins) as far as I understand. I don't know if boost is installed automatically or someone would have to do it manually.

Comment (elsagermann) [reply env.tsa.sh]: Ok, it's fine then, was just wondering if you planned to merge before the path was made, but shouldn't be the case if I understand well.

**Pull request merged**